tests/RuleEvaluatorTest.php

```diff
@@ -79,6 +97,14 @@ public function testReturnsTrueIfAttributesMatchAndConditions() {
         );
     }
 
+    public function testReturnsTrueIfAttributesMatchSemverConditions() {
+        $rules = [$this->semverRule];
+        $this->assertEquals(
+            RuleEvaluator::findMatchingRule(['appVersion' => '1.5.0'], $rules),
+            $this->semverRule
```

Comment:
While I know this will pass now, just to make sure we're doing more than string comparison, let's make the test data a bit trickier, e.g., comparing `1.12.0` to `1.2.0` or something.

Reply:
I don't follow how this example exercises the functionality different.